defaults/main.yml

```diff
@@ -319,6 +320,15 @@ postgresql_wal_receiver_timeout: 60s
 # time to wait before retrying to retrieve WAL after a failed attempt
 postgresql_wal_retrieve_retry_interval: 5s # (>= 9.5)
 
+# - Subscribers - (>= 10) 
+
+# These settings are ignored on a publisher.
+
+postgresql_max_logical_replication_workers: 4	# (>= 10) taken from max_worker_processes
```

Comment:
The tabs here and on the changes throw a syntax error for me

Reply:
Hi Ben,
I removed the tabs and hope it now works. Strangley on my system (Ubuntu 16.04/ansible 2.4.2.0) I hab no trouble.